docs/graphql/manual/guides/integrations/index.rst

```diff
@@ -3,15 +3,20 @@ Guides: Integration/migration tutorials
 
 Articles:
 ^^^^^^^^^
+
 - `Blog: Move from firebase to realtime GraphQL on Postgres <https://blog.hasura.io/firebase2graphql-moving-from-firebase-to-realtime-graphql-on-postgres-4d36cb7f4eaf>`__.
 - `Blog: Create a Gatsby site using GraphQL on Postgres <https://blog.hasura.io/create-gatsby-sites-using-graphql-on-postgres-603b5dd1e516>`__.
 - `Blog: Instant GraphQL on AWS RDS <https://blog.hasura.io/instant-graphql-on-aws-rds-1edfb85b5985>`__.
 - `Blog: Using TimescaleDB with Hasura GraphQL <https://blog.hasura.io/using-timescaledb-with-hasura-graphql-d05f030c4b10>`__.
 - :doc:`Auth0 JWT Integration with Hasura GraphQL <auth0-jwt>`
 
+Integrations:
```

Comment:
Remove this section. Add the link under the list above as `` :doc:`subscriptions` ``